acceptance-tests/src/test/java/tech/pegasys/signers/dsl/hashicorp/HashicorpVaultDockerCertificate.java

```diff
@@ -53,16 +54,19 @@ private HashicorpVaultDockerCertificate(
     this.tlsPrivateKey = tlsPrivateKey;
   }
 
-  public static HashicorpVaultDockerCertificate create() {
+  public static HashicorpVaultDockerCertificate create(
+      final SelfSignedCertificate selfSignedCertificate) {
     try {
-      final SelfSignedCertificate selfSignedCertificate = SelfSignedCertificate.generate();
       final Path certificateDirectory = createDestinationCertificateDirectory();
-      final Path tlsCertificate =
-          copyCertificate(selfSignedCertificate.certificatePath(), certificateDirectory);
-      final Path tlsPrivateKey =
-          copyCertificate(selfSignedCertificate.privateKeyPath(), certificateDirectory);
```

Comment:
Just a side note, I think docker-java API also provide support for "copyFileIntoContainer". Not as part of this PR, but in future we might be able to get away with all this fidling with mounting files if we copy the generated certificate into the container before starting hashicorp.